src/fu-util.c

```diff
@@ -1250,19 +1269,17 @@ static gboolean
 fu_util_get_results (FuUtilPrivate *priv, gchar **values, GError **error)
 {
 	g_autofree gchar *tmp = NULL;
-	g_autoptr(FwupdDevice) dev = NULL;
+	g_autoptr(FwupdDevice) indev = NULL;
```

Comment:
what does `indev` mean?

Reply:
I just didn't want to reuse the g_autoptr twice. Will it handle freeing in between uses? If so I'll just keep it as one pointer.

Reply:
Nahh, the g_autoptr is fine, I just didn't like the indev name.